src/components/activity/ActivityFormPanel.svelte

```diff
@@ -167,51 +193,104 @@
 
   <svelte:fragment slot="body">
     {#if $selectedActivity}
-      <fieldset>
-        <label for="id">Activity ID</label>
-        <input class="st-input w-100" disabled name="id" value={id} />
-      </fieldset>
+      <div class="activity-header">
+        <div class="activity-header-title-placeholder st-typography-medium">
+          {activityName}
```

Comment:
@lklyne @camargo any thoughts on what nameless activities should have as a title? Currently going with "Unnamed Activity" and the user won't be able to edit the name.
<img width="864" alt="image" src="https://user-images.githubusercontent.com/4419607/184963864-a0e2cd06-06f7-4af0-8141-c1f2ffc661b3.png">

Reply:
Nvm resolved this with Lyle – we'll show activity type.